_layouts/default.html

```diff
@@ -20,11 +20,21 @@
     <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.url }}">
   </head>
   <body>
+    <main class="bg-white w-main">
+
     {% include nav.html %}
-    <main>
-      {{ content }}
-    </main>
+    {{ content }}
     {% include footer.html %}
+    </main>
+    <div class="bg-element fixed top-0 h-100 bg-accent right-0 overflow-hidden">
+      <div class="absolute mono text-right bottom-0 right-0">
+        <div class="container">
+          <h1 class="heading-2" id="clock-element">00:00:00</h1>
+          <h2 class="heading-3 mb-0">UTC-04:00</h2>
```

Comment:
```suggestion
          <h2 class="heading-3 mb-0">UTC-04</h2>
```
Don't think you have the :00 in UTC times, they are usually natural numbers

Reply:
also I thought you were more in favour of UTC-4?